docs/wiki/Whats-new.md

```diff
@@ -3,22 +3,88 @@
 - [In this Section](#in-this-section)
 - [Updates](#updates)
   - [November 2022](#november-2022)
+    - [Docs](#docs)
+    - [Tooling](#tooling)
+  - [Policy](#policy)
+  - [Other](#other)
```

Comment:
Could you revert all of these changes back. Suspect it's an extension that's done this 👎

We just want the month headings in the TOC